diff -Naur gsettings-desktop-schemas-3.20.0.orig/schemas/gsettings-desktop-schemas.convert gsettings-desktop-schemas-3.20.0/schemas/gsettings-desktop-schemas.convert
--- gsettings-desktop-schemas-3.20.0.orig/schemas/gsettings-desktop-schemas.convert	2016-06-16 21:02:34.177861425 +0700
+++ gsettings-desktop-schemas-3.20.0/schemas/gsettings-desktop-schemas.convert	2016-06-16 21:08:58.683980558 +0700
@@ -1,5 +1,6 @@
 # From org.gnome.desktop.background.gschema.xml.in
-# [org.gnome.desktop.background]
+[org.gnome.desktop.background]
+picture-uri = /desktop/gnome/background/picture_filename
 # Disabled as we don't want to migrate the default background
 # settings as we can't convert the picture filename
 
diff -Naur gsettings-desktop-schemas-3.20.0.orig/schemas/org.gnome.desktop.background.gschema.xml.in gsettings-desktop-schemas-3.20.0/schemas/org.gnome.desktop.background.gschema.xml.in
--- gsettings-desktop-schemas-3.20.0.orig/schemas/org.gnome.desktop.background.gschema.xml.in	2016-06-16 21:02:30.785772146 +0700
+++ gsettings-desktop-schemas-3.20.0/schemas/org.gnome.desktop.background.gschema.xml.in	2016-06-16 21:06:16.907723322 +0700
@@ -10,7 +10,7 @@
       </description>
     </key>
     <key name="picture-options" enum="org.gnome.desktop.GDesktopBackgroundStyle">
-      <default>'zoom'</default>
+      <default>'stretched'</default>
       <summary>Picture Options</summary>
       <description>
         Determines how the image set by wallpaper_filename is rendered.
@@ -19,7 +19,7 @@
       </description>
     </key>
     <key name="picture-uri" type="s">
-      <default>'file://@datadir@/backgrounds/gnome/adwaita-timed.xml'</default>
+      <default>'file://@datadir@/gnome-background-properties/gnome-background-properties/blankon-wallpapers.xml'</default>
       <summary>Picture URI</summary>
       <description>
         URI to use for the background image. Note that the backend only supports
diff -Naur gsettings-desktop-schemas-3.20.0.orig/schemas/org.gnome.desktop.interface.gschema.xml.in gsettings-desktop-schemas-3.20.0/schemas/org.gnome.desktop.interface.gschema.xml.in
--- gsettings-desktop-schemas-3.20.0.orig/schemas/org.gnome.desktop.interface.gschema.xml.in	2016-06-16 21:02:34.177861425 +0700
+++ gsettings-desktop-schemas-3.20.0/schemas/org.gnome.desktop.interface.gschema.xml.in	2016-06-16 21:06:14.827668582 +0700
@@ -97,14 +97,14 @@
       </description>
     </key>
     <key name="icon-theme" type="s">
-      <default>'Adwaita'</default>
+      <default>'Tebu-Flat'</default>
       <summary>Icon Theme</summary>
       <description>
         Icon theme to use for the panel, nautilus etc.
       </description>
     </key>
     <key name="gtk-theme" type="s">
-      <default>'Adwaita'</default>
+      <default>'Bromo'</default>
       <summary>Gtk+ Theme</summary>
       <description>
         Basename of the default theme used by gtk+.
@@ -118,7 +118,7 @@
       </description>
     </key>
     <key name="font-name" type="s">
-      <default>'Cantarell 11'</default>
+      <default>'Aurulent Sans 10'</default>
       <summary>Default font</summary>
       <description>
         Name of the default font used by gtk+.
@@ -162,14 +162,14 @@
       </description>
     </key>
     <key name="document-font-name" type="s">
-      <default>'Sans 11'</default>
+      <default>'Sans 10'</default>
       <summary>Document font</summary>
       <description>
         Name of the default font used for reading documents.
       </description>
     </key>
     <key name="monospace-font-name" type="s">
-      <default>'Monospace 11'</default>
+      <default>'Monospace 10'</default>
       <summary>Monospace font</summary>
       <description>
         Name of a monospaced (fixed-width) font for use in locations like
@@ -200,7 +200,7 @@
       </description>
     </key>
     <key name="cursor-theme" type="s">
-      <default>'Adwaita'</default>
+      <default>'DMZ-White'</default>
       <summary>Cursor theme</summary>
       <description>Cursor theme name. Used only by Xservers that support the Xcursor extension.</description>
     </key>
diff -Naur gsettings-desktop-schemas-3.20.0.orig/schemas/org.gnome.desktop.sound.gschema.xml.in gsettings-desktop-schemas-3.20.0/schemas/org.gnome.desktop.sound.gschema.xml.in
--- gsettings-desktop-schemas-3.20.0.orig/schemas/org.gnome.desktop.sound.gschema.xml.in	2016-06-16 20:58:18.235124468 +0700
+++ gsettings-desktop-schemas-3.20.0/schemas/org.gnome.desktop.sound.gschema.xml.in	2016-06-16 21:06:49.432579263 +0700
@@ -7,7 +7,7 @@
       <description>Whether to play sounds on user events.</description>
     </key>
     <key name="theme-name" type="s">
-      <default>'freedesktop'</default>
+      <default>'blankon'</default>
       <summary>Sound theme name</summary>
       <description>The XDG sound theme to use for event sounds.</description>
     </key>
diff -Naur gsettings-desktop-schemas-3.20.0.orig/schemas/org.gnome.desktop.wm.preferences.gschema.xml.in gsettings-desktop-schemas-3.20.0/schemas/org.gnome.desktop.wm.preferences.gschema.xml.in
--- gsettings-desktop-schemas-3.20.0.orig/schemas/org.gnome.desktop.wm.preferences.gschema.xml.in	2016-06-16 20:58:18.235124468 +0700
+++ gsettings-desktop-schemas-3.20.0/schemas/org.gnome.desktop.wm.preferences.gschema.xml.in	2016-06-16 21:08:04.234547732 +0700
@@ -25,7 +25,7 @@
        </description>
     </key>
     <key name="button-layout" type="s">
-      <default>'appmenu:close'</default>
+      <default>':minimize,maximize,close'</default>
       <summary>Arrangement of buttons on the titlebar</summary>
       <description>
         Arrangement of buttons on the titlebar. The value should be a string,
@@ -174,7 +174,7 @@
       </description>
     </key>
     <key name="theme" type="s">
-      <default>'Adwaita'</default>
+      <default>'Bromo'</default>
       <summary>Current theme</summary>
       <description>
         The theme determines the appearance of window borders, titlebar,
@@ -192,7 +192,7 @@
       </description>
     </key>
     <key name="titlebar-font" type="s">
-      <default>'Cantarell Bold 11'</default>
+      <default>'Aurulent Sans Bold 10'</default>
       <summary>Window title font</summary>
       <description>
         A font description string describing a font for window titlebars.
